app/src/main/java/com/example/bigowlapp/utils/SmsSender.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bigowlapp.utils;
+
+import android.content.Context;
+import android.telephony.SmsManager;
+
+
+public class SmsSender {
+
+    private static final String SMS_RECEIVED = "android.provider.Telephony.SMS_RECEIVED";
+
+    public SmsSender() {
```

Comment:
Is it necessary to keep this here? its empty constructor

Reply:
Yes, it is a util class